docs/book/src/multiversion-tutorial/testdata/project/internal/controller/cronjob_controller.go

```diff
@@ -96,6 +97,7 @@ var (
 //
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.19.1/pkg/reconcile
+// nolint:gocyclo
```

Comment:
Can you rebase your PR with master ?
I think it is added in the default scaffold now